client/src/components/CCDLDatasetDownloadStarted.js

```diff
@@ -13,13 +13,18 @@ import DownloadSVG from '../images/download-folder.svg'
 // View when the download should have been initiated
 export const CCDLDatasetDownloadStarted = () => {
   // open the file in a new tab
-  const { downloadableDataset } = useCCDLDatasetDownloadModalContext()
+  const { downloadableDataset, modality } = useCCDLDatasetDownloadModalContext()
   const { size: responsiveSize } = useResponsive()
   const links = downloadableDataset.ccdl_project_id
     ? projectLinks
     : portalWideLinks
   const { learnMore } = links[downloadableDataset.ccdl_name]
 
+  const getReadableDataFormat = () => {
+    if (modality === 'SPATIAL') return getReadable('SPATIAL_SPACERANGER')
```

Comment:
```suggestion
    if (downloadableDataset.ccdl_modality === 'SPATIAL') return getReadable('SPATIAL_SPACERANGER')
```

Source of truth should just be the downloadable dataset